sys/include/net/ipv6/addr.h

```diff
@@ -729,6 +729,18 @@ char *ipv6_addr_to_str(char *result, const ipv6_addr_t *addr, uint8_t result_len
  */
 ipv6_addr_t *ipv6_addr_from_str(ipv6_addr_t *result, const char *addr);
 
+/**
+ * @brief split IPv6 address string representation and return remaining string
+ *
+ * @note Will change @p separator position in @p addr_str to '\0'
+ *
+ * @param[in,out]   addr_str    Address to split
+ * @param[in]       separator   Separator char to use
+ *
+ * @return      pointer to the remaining of the string
```

Comment:
It's implicit with this wording, but maybe bit more specific here

```suggestion
 * @return      string following the first occurrence of @p separator in @p addr_str
 * @return      "" if @p separator does not appear in @p addr_str
```

(please adapt line length, I did not check if it is <80 chars)

Reply:
The alternative case here is still missing ;-).